src/storage/src/hummock/store/version.rs

```diff
@@ -531,13 +531,14 @@ impl HummockVersionReader {
 const SLOW_ITER_FETCH_META_DURATION_SECOND: f64 = 5.0;
 
 impl HummockVersionReader {
-    pub async fn get(
+    pub async fn get<O>(
```

Comment:
(Not an issue. Why it is O here. 🥵

Reply:
It refers to the output type of `get`. For example, the current implementation in main branch, the `on_key_value_fn` will be
```
|key: FullKey<&[u8]>, value: &[u8]| -> Bytes {
    Bytes::copy_from_slice(value)
}
```
which creates a Bytes from the value u8 slice, and in this implementation the `O` is `Bytes`.